### ISSUE TITLE
exitEditMode doesn't work

### ISSUE THREAD

**Issue by syabro** [iddan/react-spreadsheet#283]
I'm trying to finish edit mode when  react-select is closing menu. 
Code is from storybook,  I just added `exitEditMode`
But looks like `exitEditMode()` it's being ignored...

```ts
export const SelectEdit: DataEditorComponent<Cell> = ({ cell, onChange, exitEditMode }) => {
  const handleChange = React.useCallback(
    (selection) => {
      onChange({ ...cell, value: selection ? selection.value : null });
    },
    [cell, onChange]
  );
  const option = React.useMemo(
    () => cell && OPTIONS.find((option) => option.value === cell.value),
    [cell]
  );
  return (
    <Select
      value={option}
      onChange={handleChange}
      options={OPTIONS}
      autoFocus
      defaultMenuIsOpen
      onMenuClose={() => exitEditMode()}  // ADDED HERE
    />
  );
};
```

**Comment by iddan** [comment]
Hey, thank you for the detailed reporting. Can you please provide a reproduction codesanbox?

**Comment by syabro** [comment]
@iddan sure!
https://codesandbox.io/s/patient-cache-1t2kwj

**Comment by iddan** [comment]
Published a fix in + react-spreadsheet@0.7.8

**Comment by syabro** [comment]
Works! Thank you!
